test/e2e/examples.go

```diff
@@ -42,7 +42,7 @@ const (
 	serverStartTimeout = framework.PodStartTimeout + 3*time.Minute
 )
 
-var _ = framework.KubeDescribe("[Feature:Example]", func() {
+var _ = framework.SIGDescribe("node", "[Feature:Example]", func() {
```

Comment:
I think that instead of changing anything here, we should probably move this whole file to tests/e2e/node.

@spiffxp

Reply:
yes, will update the PR.

Reply:
Scoped the PR to only move `test/e2e/examples.go` as the other changes were updating `conformance.yaml`, will raise a separate PR for it.